README.rst

```diff
@@ -48,27 +48,41 @@ in the ``settings.py`` of your Scrapy project.
 
 You also need to set the ``ZYTE_API_KEY``.
 
-Lastly, make sure to `install the asyncio-based Twisted reactor
+Make sure to `install the asyncio-based Twisted reactor
 <https://docs.scrapy.org/en/latest/topics/asyncio.html#installing-the-asyncio-reactor>`_
 in the ``settings.py`` file as well.
 
+If you are using Scrapy 2.7 or later, also set the
+``REQUEST_FINGERPRINTER_CLASS`` setting to
+``"scrapy_zyte_api.ScrapyZyteAPIRequestFingerprinter"`` to make `request
+fingerprinting <https://docs.scrapy.org/en/latest/topics/request-response.html#request-fingerprints>`_
+take Zyte API parameters into account.
+
 Here's an example of the things needed inside a Scrapy project's ``settings.py`` file:
 
 .. code-block:: python
 
     DOWNLOAD_HANDLERS = {
-        "http": "scrapy_zyte_api.handler.ScrapyZyteAPIDownloadHandler",
-        "https": "scrapy_zyte_api.handler.ScrapyZyteAPIDownloadHandler"
+        "http": "scrapy_zyte_api.ScrapyZyteAPIDownloadHandler",
+        "https": "scrapy_zyte_api.ScrapyZyteAPIDownloadHandler"
     }
 
     # Having the following in the env var would also work.
     ZYTE_API_KEY = "<your API key>"
 
     TWISTED_REACTOR = "twisted.internet.asyncioreactor.AsyncioSelectorReactor"
 
+    # Only on Scrapy 2.7 or later:
+    REQUEST_FINGERPRINTER_CLASS = "scrapy_zyte_api.ScrapyZyteAPIRequestFingerprinter"
+
 The ``ZYTE_API_ENABLED`` setting, which is ``True`` by default, can be set to
 ``False`` to disable this plugin.
 
+If you had a value for the ``REQUEST_FINGERPRINTER_CLASS`` previously set,
+assign that value to the ``ZYTE_API_FALLBACK_REQUEST_FINGERPRINTER_CLASS``
```

Comment:
Let's show an example on how to assign this to `ZYTE_API_FALLBACK_REQUEST_FINGERPRINTER_CLASS` so that users can easily copy it from the code block above.